chia/wallet/did_wallet/did_wallet.py

```diff
@@ -16,6 +17,9 @@
 from chia.util.ints import uint64, uint32, uint8
 from chia.wallet.util.transaction_type import TransactionType
 
+if TYPE_CHECKING:
+    from chia.wallet.wallet_state_manager import WalletStateManager
+
```

Comment:
I don't super care here, but I would probably default to putting this at the end of the import list.  I don't know if isort cares.

Reply:
Yeah, was a toss-up of keeping it in alphabetical order, or separating it out